Jenkinsfile

```diff
@@ -1,10 +1,22 @@
-def pod, utils
+def pod, utils, devel
 node {
     checkout scm
     pod = readFile(file: "manifests/pod.yaml")
     utils = load("utils.groovy")
+
+    // just autodetect if we're in prod or not
+    devel = (env.JENKINS_URL != 'https://jenkins-fedora-coreos.apps.ci.centos.org/')
+
+    if (devel) {
+        echo "Running in devel mode on ${env.JENKINS_URL}."
+    }
 }
 
+properties([
+    disableConcurrentBuilds(),
+    pipelineTriggers(devel ? [] : [cron("H/30 * * * *")])
```

Comment:
The H/30 makes it run every 30 minutes but at indeterminate times, right? Is that what we want. Sometimes I like knowing I can expect a build to kick off at the 0 or 30.

Reply:
Yeah, the idea is that you spread the load from all the other jobs across the hour instead of having a bunch of jobs all running at 0 and 30. OTOH... we own the Jenkins master, and there's only one job... But then again, we don't own the cluster and there might be *many* Jenkins masters, so I feel like that argument is still somewhat relevant.

> Sometimes I like knowing I can expect a build to kick off at the 0 or 30.

Have some tea and read emails, or really... just click the "Build Now" button :)